include/alpaka/KernelBundle.hpp

```diff
@@ -28,23 +28,35 @@ namespace alpaka
         using ArgTuple = std::tuple<remove_restrict_t<std::decay_t<TArgs>>...>;
 
         // Constructor
-        constexpr KernelBundle(KernelFn kernelFn, TArgs&&... args)
-            : m_kernelFn{std::move(kernelFn)}
+        constexpr KernelBundle(KernelFn const& kernelFn, TArgs&&... args)
+            : m_kernelFn{kernelFn}
             , m_args(std::forward<TArgs>(args)...)
         {
         }
 
         constexpr KernelBundle(KernelBundle const& b) = default;
+        constexpr KernelBundle& operator=(KernelBundle const&) = default;
 
+        /** allow move assignment and constriction
+         *
+         *  @attention if the functor or the arguments contains non movable types the move operators can be
+         * inaccessible.
+         *
+         *  @{
+         */
         constexpr KernelBundle(KernelBundle&& b) = default;
+        constexpr KernelBundle& operator=(KernelBundle&&) = default;
+
+        /** @} */
 
         constexpr auto operator()(auto const& acc) const
         {
             std::apply([&](auto const&... args) constexpr { m_kernelFn(acc, args...); }, m_args);
         }
 
         KernelFn const m_kernelFn;
```

Comment:
Should be not const, because with const member variables, the KernelBundle is not moveable anymore.